### ISSUE TITLE
Disaggregations appear duplicate in IP PRP

### ISSUE THREAD

**Issue by KKaselkar** [unicef/etools-partner-reporting-portal#943]
dev.prp environment 
Iraq workspace
PCA 2016/04/PD2018454

disaggregations appear multiple times in IP reporting 
screenshot below

https://dev.partnerreportingportal.org/app/IRQ/ip-reporting/pd/948/report/215/edit?page=1

**Comment by KKaselkar** [comment]
![image](https://user-images.githubusercontent.com/32198614/46419120-34385c80-c6fb-11e8-9b1a-e2a1d23fe9b5.png)

![image](https://user-images.githubusercontent.com/32198614/46419147-4914f000-c6fb-11e8-9509-f7ec4f47020f.png)

**Comment by n1207n** [comment]
I noticed that Age disaggregation had 2 Values without external_id. This means that someone created these in PRP manually. I have deleted them, so this should not happen again.

![Screen Shot 2018-10-08 at 1.46.23 PM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/58d2f674dc6eec1a00ff72e7/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5710b76236454fb6b7191a379f1162d4f5d5c5ecc3c4a4356bcbbd671f3e57862804e0ee5823b5777933d12aab1d11eba01ba29894a82b0bb884ad2f42f67b6d5db1063474f4dcbed5727a19e41c95df542e14e54a2133e66e1772431.png)
